### ISSUE TITLE
Those bouncing balls?

### ISSUE THREAD

**Issue by original-dirigible** [thetwom/toc2#18]
I really miss the animation of those bouncing balls.  
Please consider putting them back in.  
Love your app.

**Comment by thetwom** [comment]
Thanks for your interest in the app and giving feedback. That's worth a lot!

I also liked the bouncing balls better, so let me explain, why I replaced them with the animation on the screen bottom.

The bouncing balls were drawn over thea area where we change the speed. So each time, the position of a ball changes, the whole speedchanging area also has to be redrawn (plus everything which is in its rectangular bounding box). This has quite some effect on the performance (and I guess it would also use more battery, however I never really measured). In contrast, in the new animation only the few pixels on the bottom have to be redrawn. This is much cheaper. 

So, in summary, I treated the bouncing balls for performance, hoping that the new speed effect is giving the same amount of visible feedback.

I hope, this helps understanding why I don't plan to bring the balls back. 

Please keep up posting any feedback, questions, suggestions or bugs!

**Comment by thetwom** [comment]
Closing this now, since this will not come back due to the mentioned reasons. Please feel free to suggest alternatives by opening a new issue.
